src/amo/components/HeroRecommendation/index.js

```diff
@@ -199,9 +136,13 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
     // translators: If uppercase does not work in your locale, change it to lowercase.
     // This is used as a secondary heading.
     const recommended = i18n.gettext('RECOMMENDED');
+    const classnames = makeClassName(
+      'HeroRecommendation',
+      `${gradient.start}-${gradient.end}`,
```

Comment:
I'm not sure of any way we can share the knowledge of the classNames between the stylesheet and the component code. If there is a way, then I agree that that would be a great thing to check, but if there isn't then it would mean duplicating the classNames in both places, and I fear that's not worth it.

For now I'm going to simply log a message to the console as a hint in case there are css issues when running this in the future. Maybe that's enough?